src/main.ts

```diff
@@ -57,10 +58,48 @@ const mainExecution = (program) => {
     /**
      * Push section
      */
-    const _push = (forceFrom: string|null, callback: ErrOnlyCallback): void => {
+    const _push = (forceFrom: string | null, callback: ErrOnlyCallback): void => {
         push(forceFrom || program.push, program.source, program.deploy, callback);
     };
 
+    /**
+     * Push multiple packages
+     */
+    const _multiplePush = (callback: ErrOnlyCallback): void => {
+        const fromPath = program.push;
```

Comment:
If you're thinking about where this method is, where it's used and what it does, I was really thinking that `multiplePush` is a good name. 😉